ecs-init/exec/iptables/iptables.go

```diff
@@ -29,10 +31,16 @@ const (
 	credentialsProxyPort          = "80"
 	localhostIpAddress            = "127.0.0.1"
 	localhostCredentialsProxyPort = "51679"
+	localhostNetwork              = "127.0.0.0/8"
 	// iptablesAppend enumerates the 'append' action
 	iptablesAppend iptablesAction = "-A"
+	// iptablesAppend enumerates the 'insert' action
```

Comment:
nit: should this comment refer to  `iptablesInsert`?

Reply:
good catch, fixed